samples/bluetooth/periodic_adv/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _bluetooth-periodic-advertising-sample:
+.. zephyr:code-sample:: ble_periodic_adv
+   :name: Periodic Advertising
+   :relevant-api: bt_gap bluetooth
 
-Bluetooth: Periodic Advertising
-###############################
+   Use BLE Periodic Advertising functionality.
```

Comment:
As an overall comment we perhaps should refrain from using `BLE` in our documentation. It's not an adapted acronym for Bluetooth Low Energy, and IIRC the Bluetooth SIG actually discourages the use of it. Suggest to write `Bluetooth LE` or `Bluetooth Low Energy` as an alternative

Reply:
Agreed. I've addressed the BLE->Bluetooth LE rename across samples in an additional commit in this PR